alma/lib/Utils/functions.php

```diff
@@ -150,15 +150,31 @@ function almaFormatPrice($cents, $id_currency = null)
 function getDateFormat($locale, $timestamp)
 {
     try {
+        if (!class_exists(IntlDateFormatter::class)) {
+            throw new Exception('IntlDateFormatter: is not available');
```

Comment:
please, don't use the `throw` then `catch` exception pattern.
if class exists, just do what you have to do and `return`
if class does not exits, then `return getFrenchDateFormat($timestamp);`